features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/data/AuthManager.jsx

```diff
@@ -177,8 +177,13 @@ class AuthManager {
         if (user) {
             const userScopes = user.scopes;
             const validScope = APIClient.getScopeForResource(resourcePath, resourceMethod);
-            return validScope.then((scope) => {
-                return userScopes.includes(scope);
+            let validity = []
+            return validScope.then((scopes) => {
+                scopes.map(scope => {
+                    const isValid = userScopes.includes(scope);
+                    validity.push(isValid)
+                });
+                return validity.includes(false) ? false : true
```

Comment:
This is AND logic right ? But scopes in REST API has OR meaning , I:e `apim:api_create,apim:api_delete` means at least one of the @malinthaprasan correct me if I'm wrong ?

Reply:
Hmm, yes. I think we should check for OR here. 
```
return validity.includes(true) ? true : false
```